pkg/roll/execute_test.go

```diff
@@ -33,7 +34,7 @@ func TestSchemaIsCreatedAfterMigrationStart(t *testing.T) {
 		ctx := context.Background()
 		version := "1_create_table"
 
-		if err := mig.Start(ctx, &migrations.Migration{Name: version, Operations: migrations.Operations{createTableOp("table1")}}); err != nil {
+		if err := mig.Start(ctx, &migrations.Migration{Name: version, Operations: migrations.Operations{createTableOp("table1")}}, backfill.NewConfig()); err != nil {
```

Comment:
The signature of `Start` could change to:

```go
func (m *Roll) Start(ctx context.Context, migration *migrations.Migration, c ...*backfill.Config) error
```

so that the config becomes optional and we don't have to do this everywhere.

It's arguably an anti-pattern though. WDYT?